assets/js/vue/components/data-land/HelpModal.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <span>
+    <b-button class="sm rounded-circle" @click="openHelpModal" variant="primary">
+      <i class="fa fa-question-circle"></i>
+    </b-button>
+    <b-modal
+        id="modal-1"
+        title="Alternative Data Access Methods"
+        v-model="showHelpDialog"
+        :static="staticValue"
+        ok-variant="primary"
+        hide-footer
+        header-text-variant="dark">
+      <div class="modal-body">
```

Comment:
```suggestion
      <div class="modal-body" align="left">
```
This works, but I'm not sure if this is proper.

Reply:
you wanna use `text-left` in class